base-system/usrroot/etc/default/earlyoom

```diff
@@ -0,0 +1,25 @@
+# Default settings for earlyoom. This file is sourced by /bin/sh from
+# /etc/init.d/earlyoom or by systemd from earlyoom.service.
+
+# Options to pass to earlyoom
+# Report memory status every 15 seconds
+# SIGTERM if memory below 20
+# SIGKILL if memory below 15
+# Send bus notifications (for GUI notifications)
+EARLYOOM_ARGS="-r 15 -m 20,15 -n"
+
+# Examples:
+
+# Print memory report every minute instead of every hour
+# EARLYOOM_ARGS="-r 60"
```

Comment:
Let's take a note on this reports. Might be good to consider for the future manager dashboard UI to get the status of the computers.

Reply:
Sounds good